### ISSUE TITLE
Update comment parser to read rule slashes

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#1116]
The parsing of JSON configuration comments like `/*eslint */` doesn't currently work if a rule has a slash in it. Update the parser so that this works correctly:

``` js
/*eslint plugin-name/rule-id: 2
```

The rule should be identified as `plugin-name/rule-id`. This is in the `parseJsonConfig()` function inside of `eslint.js`.
